resources/cloudwatchlogs-loggroups.go

```diff
@@ -130,8 +135,13 @@ func (f *CloudWatchLogsLogGroup) Properties() types.Properties {
 		Set("CreatedTime", f.logGroup.CreationTime).
 		Set("LastEvent", f.lastEvent)
 
+	if f.logGroup.RetentionInDays != nil {
```

Comment:
You don't need to guard this with an if statement, the `Set` will handle all values properly including a nil.